core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -970,6 +963,48 @@ private void initAfterRebalance() {
             return new AbstractMap.SimpleImmutableEntry<>(bb, currentHandle);
         }
 
+        /**
+         * Advances next to higher entry without creating a ByteBuffer for key.
+         * Return previous index
+         */
+        Handle advanceStream(OakRKeyReferenceImpl key) {
+
+            if (state == null) {
+                throw new NoSuchElementException();
+            }
+
+            Chunk.State chunkState = state.getChunk().state();
+
+            if (chunkState == Chunk.State.RELEASED) {
+                initAfterRebalance();
+            }
+
+            if (key != null) state.getChunk().setKeyRefer(state.getIndex(),key);
```

Comment:
In a  block please ...

Reply:
This and other comments that are resolved without comment, means they are fixed